### PR TITLE
Remove GCC 4.8 support

### PR DESCRIPTION
**Pull request by changlan**
GCC 4.8 uses a different ABI. As a result, gRPC built by GCC 4.8 doesn't work in any case (#532). In addition, if gRPC is built by GCC 5+, BESS cannot be built by GCC 4.8. This PR ditches the support for GCC 4.8.

### CODE REVIEW

File: core/Makefile [hunk 1]
```diff
@@ -21,8 +21,8 @@ CXXCOMPILER := $(shell expr $(word 1, $(shell $(CXX) --version)) : '\(clang\|g++
 CXXVERSION := $(shell $(CXX) -dumpversion)
 
 ifeq "$(CXXCOMPILER)" "g++"
-ifneq "$(shell printf '$(CXXVERSION)\n4.7' | sort -V | head -n1)" "4.7"
-    $(error g++ 4.8 or higher is required)
+ifneq "$(shell printf '$(CXXVERSION)\n5' | sort -V | head -n1)" "5"
+    $(error g++ 5 or higher is required)
```
Comment (sangjinhan) [inline core/Makefile]: We can recommend `container_build.py` here..?

Comment (changlan) [inline core/Makefile]: Sure.

Comment (changlan) [inline core/Makefile]: Done

**Comment by barath** [comment]
photo on 2014-08-21 at 18 39 2

**Pull request merged**